infrastructure/stacks/roles/locals.tf

```diff
@@ -0,0 +1,15 @@
+locals {
+
+
+  fuzzy-search-dynamodb_access_iam_name = "${var.service_tag}-${var.profile}-${var.project_tag}-dynamodb-access"
```

Comment:
Needs to align with the K8s service account name.  And please call this service-account-role or service-account-iam-role